File: Mobil/app/src/main/java/com/example/petalarmmobil/MainActivity.java
package com.example.petalarmmobil;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
    public void ingresar(View view){
        Intent intent = new Intent(this, MainMenu.class);
        startActivity(intent);
    }
    public void registrarse(View view){
        Intent intent = new Intent(this, Registro.class);
        startActivity(intent);
    }
}

File: Mobil/app/src/main/java/com/example/petalarmmobil/MainActivity.kt
package com.example.petalarmmobil

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class MainActivity : AppCompatActivity() {

    private lateinit var btnRegistro: Button

    private lateinit var btnIngresar: Button
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


        btnRegistro = findViewById(R.id.btn_registrarse)
        btnIngresar = findViewById(R.id.btn_ingresar)

        btnRegistro.setOnClickListener {
            setContentView(R.layout.activity_registro)
        }


        btnIngresar.setOnClickListener {
            setContentView(R.layout.activity_main_menu)
        }



    }
}